tools/start_server.py

```diff
@@ -319,6 +321,9 @@ def main():
 
   host = str(sys.argv[1])
   port = int(sys.argv[2])
+  WebEngineManager.WEBENGINE_REMOTE_HOST = str(sys.argv[3]) if (len(sys.argv) > 3 and str(sys.argv[3]).strip() != "") else host
```

Comment:
```suggestion
  WebEngineManager.WEBENGINE_REMOTE_HOST = str(sys.argv[3]) if len(sys.argv) > 3 else host
```

is this really necessary? Someone would have to put whitespace in quotes in order to hit this code

Reply:
good catch. That check wouldn't work as expected